CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### 0.2.0 (Next)
 
 * [#13](https://github.com/dblock/danger-changelog/pull/13): Fix: suggested CHANGELOG entry may not be of correct format - [@dblock](https://github.com/dblock).
+* [#17](https://github.com/dblock/danger-changelog/pull/17): Fix: improve CHANGELOG line validation - [@antondomashnev](https://github.com/antondomashnev).
```

Comment:
Maybe something more explicit like "handle changelog lines that mistakenly start with a symbol other than *"?